docs/content/reference/api-usage.md

```diff
@@ -0,0 +1,49 @@
+---
+title: NGINX Service Mesh API
+description: "Instructions for using the NGINX Service Mesh API."
+toc: true
+tags: ["api"]
+weight: 200
+categories: ["reference"]
+docs: "DOCS-702"
+---
+
+## Overview
+
+The NGINX Service Mesh API lives in a Kubernetes Custom Resource which allows a way to manage the global mesh configuration. This resource is created when the mesh is deployed, and can be updated at runtime via the Kubernetes API.
+
+## Modifying the global mesh configuration
+
+To update the global mesh configuration, use `kubectl` to edit the `meshconfig` resource that lives in the NGINX Service Mesh namespace. By default, the name of the resource is `nginx-mesh-config`.
+
+```bash
+kubectl edit meshconfig nginx-mesh-config -n nginx-mesh
+```
+
+This will open your editor and allow for changes. To see the configurable fields, download the custom resource definition:
```

Comment:
```suggestion
This will open your default text editor and allow for changes. To see the configurable fields, download the custom resource definition:
```

Adding slightly more specificity. In context I presume this means `vi` or `nano` depending on the distribution.